### ISSUE TITLE
Droplet packaging fails after push

### ISSUE THREAD

**Issue by stefanschneider** [cloudfoundry/cloud_controller_ng#462]
When pushing an asp.net app with tfs files inside ( zip file that fails at push: https://drive.google.com/file/d/0B-YeJy5DlfNKc0pFRjNyUXM4dXc/view?usp=sharing ) CC will just fail to package the app.

Error from cf push:
```
cf push tfst2
Updating app tfst2 in org diego / space diego as admin...
OK

Uploading tfst2...
Uploading app files from: C:\Users\stefans\Documents\Visual Studio 2015\Projects\cfweb6-dup\cfweb6
Uploading 94.4M, 776 files
Done uploading
FAILED
Error uploading application.
An unknown error occurred.

```

Error from cf push with trace:
```
RESPONSE: [2015-11-20T16:21:06+02:00]
HTTP/1.1 200 OK
Content-Length: 491
Content-Type: application/json;charset=utf-8
Date: Fri, 20 Nov 2015 14:21:06 GMT
Server: nginx
X-Cf-Requestid: a453c7d0-71f9-4b0b-7e7c-08d1c7dda2b2
X-Content-Type-Options: nosniff
X-Vcap-Request-Id: b300e5ee-7b86-47e7-55fc-49dd52381fd1::4535e0fc-f16b-456c-beb9-734769b5eaa3

{
  "metadata": {
    "guid": "2a9f6bc4-d094-48cb-b8fd-919cbe5da5c2",
    "created_at": "2015-11-20T14:21:01Z",
    "url": "/v2/jobs/2a9f6bc4-d094-48cb-b8fd-919cbe5da5c2"
  },
  "entity": {
    "guid": "2a9f6bc4-d094-48cb-b8fd-919cbe5da5c2",
    "status": "failed",
    "error": "Use of entity>error is deprecated in favor of entity>error_details.",
    "error_details": {
      "error_code": "UnknownError",
      "description": "An unknown error occurred.",
      "code": 10001
    }
  }
}
FAILED
Error uploading application.
An unknown error occurred.
FAILED
Error uploading application.
An unknown error occurred.

```

Error log from CC:
```
2015-11-20 14:01:30.972471 cc.background  pid=12501 tid=37de fid=00bd jobs/exception_catching_job.rb/log_error:24    ERROR -- Request failed: 500: {"code"=>10001, "description"=>"undefined method `use_instad=' for #<MIME::Type::Columnar:0x007f3572f017a8>", "error_code"=>"CF-NoMethodError", "backtrace"=>["/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:121:in `block in load_use_instead'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:53:in `block (3 levels) in each_file_line'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:48:in `each_line'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:48:in `block (2 levels) in each_file_line'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:45:in `open'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:45:in `block in each_file_line'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:42:in `synchronize'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:42:in `each_file_line'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:120:in `load_use_instead'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/type/columnar.rb:73:in `use_instead'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/type.rb:178:in `priority_compare'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types.rb:177:in `block in type_for'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types.rb:177:in `sort'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types.rb:177:in `type_for'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types.rb:257:in `type_for'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/cloud_controller/blobstore/client.rb:66:in `block in cp_to_blobstore'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/cloud_controller/blobstore/client.rb:61:in `open'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/cloud_controller/blobstore/client.rb:61:in `cp_to_blobstore'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/cloud_controller/blobstore/client.rb:51:in `block in cp_r_to_blobstore'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/find.rb:48:in `block (2 levels) in find'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/find.rb:47:in `catch'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/find.rb:47:in `block in find'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/find.rb:42:in `each'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/find.rb:42:in `find'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/cloud_controller/blobstore/client.rb:44:in `each'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/cloud_controller/blobstore/client.rb:44:in `cp_r_to_blobstore'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/app/models/runtime/app_bits_package.rb:23:in `block in create'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/cloud_controller/blobstore/local_app_bits.rb:18:in `yield'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/cloud_controller/blobstore/local_app_bits.rb:18:in `block in from_compressed_bits'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/tmpdir.rb:88:in `mktmpdir'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/cloud_controller/blobstore/local_app_bits.rb:11:in `from_compressed_bits'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/app/models/runtime/app_bits_package.rb:20:in `create'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/app/jobs/runtime/app_bits_packer.rb:36:in `perform'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/app/jobs/wrapping_job.rb:9:in `perform'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/app/jobs/timeout_job.rb:7:in `block in perform'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/timeout.rb:88:in `block in timeout'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/timeout.rb:32:in `block in catch'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/timeout.rb:32:in `catch'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/timeout.rb:32:in `catch'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/timeout.rb:103:in `timeout'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/app/jobs/timeout_job.rb:6:in `perform'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/app/jobs/request_job.rb:13:in `perform'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/app/jobs/wrapping_job.rb:9:in `perform'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/backend/base.rb:94:in `block in invoke_job'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:61:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:61:in `block in initialize'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:66:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:66:in `execute'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:40:in `run_callbacks'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/backend/base.rb:91:in `invoke_job'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:199:in `block (2 levels) in run'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/timeout.rb:88:in `block in timeout'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/timeout.rb:98:in `call'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/timeout.rb:98:in `timeout'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:199:in `block in run'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/benchmark.rb:303:in `realtime'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:198:in `run'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:275:in `block in reserve_and_run_one_job'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:61:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:61:in `block in initialize'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:66:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:66:in `execute'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:40:in `run_callbacks'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:275:in `reserve_and_run_one_job'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:182:in `block in work_off'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:181:in `times'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:181:in `work_off'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:146:in `block (4 levels) in start'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/benchmark.rb:303:in `realtime'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:145:in `block (3 levels) in start'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:61:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:61:in `block in initialize'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:66:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:66:in `execute'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:40:in `run_callbacks'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:144:in `block (2 levels) in start'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:143:in `loop'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:143:in `block in start'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/plugins/clear_locks.rb:7:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/plugins/clear_locks.rb:7:in `block (2 levels) in <class:ClearLocks>'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:79:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:79:in `block (2 levels) in add'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:61:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:61:in `block in initialize'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:79:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:79:in `block in add'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:66:in `call'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:66:in `execute'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/lifecycle.rb:40:in `run_callbacks'", "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/delayed_job-4.0.4/lib/delayed/worker.rb:142:in `start'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/tasks/jobs.rake:40:in `start_working'", "/var/vcap/data/packages/cloud_controller_ng/7df0da82c0d720021c8dab4d510826e0eb44baa8.1-251a6a13dfb28e98725164cfe1a50c3050581d3a/cloud_controller_ng/lib/tasks/jobs.rake:11:in `block (2 levels) in <top (required)>'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/task.rb:240:in `call'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/task.rb:240:in `block in execute'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/task.rb:235:in `each'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/task.rb:235:in `execute'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/task.rb:179:in `block in invoke_with_call_chain'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/monitor.rb:211:in `mon_synchronize'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/task.rb:172:in `invoke_with_call_chain'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/task.rb:165:in `invoke'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/application.rb:150:in `invoke_task'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/application.rb:106:in `block (2 levels) in top_level'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/application.rb:106:in `each'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/application.rb:106:in `block in top_level'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/application.rb:115:in `run_with_threads'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/application.rb:100:in `top_level'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/application.rb:78:in `block in run'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/application.rb:176:in `standard_exception_handling'", "/var/vcap/packages/ruby-2.2.3/lib/ruby/2.2.0/rake/application.rb:75:in `run'", "/var/vcap/packages/ruby-2.2.3/bin/rake:33:in `<main>'"]}
      - sha1: 500fd7818df831582dffad2f51b37862af272bbd
```

We are still not sure which file causes this problem. We just found that if we comment [this line from CC]( https://github.com/cloudfoundry/cloud_controller_ng/blob/master/lib/cloud_controller/blobstore/client.rb#L66) the error does not appear anymore.

/cc @aemengo

**Comment by micahyoung** [comment]
Looking through the stacktrace, we see the following error:
```
undefined method `use_instad='' for #<MIME::Type::Columnar:0x007fb3a29ad398>\n/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:121:in `block in load_use_instead''\n/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/vendor/bundle/ruby/2.2.0/gems/mime-types-2.6.1/lib/mime/types/columnar.rb:53:in
```

It seems it's fixed in the 2.6.2 release of ruby-mime-types by this PR:
https://github.com/mime-types/ruby-mime-types/pull/109/files

Bumping the gem should be enough to fix this. 

We found it is simple to reproduce this by pushing any app with an extension for which ```MIME::Types.of(fileext)``` returns matches with a "use_instead" property set ('mdf' is one such extension)

**Comment by dieucao** [comment]
@micahyoung The gem has been bumped and this change should be available in the next cf-release.
If this doesn't address the issue for you, please open a new issue and link to this one.

https://github.com/cloudfoundry/cloud_controller_ng/commit/17a897f7f29a89394845cee608a7b29315447633
